sycl/doc/GetStartedGuide.md

```diff
@@ -44,8 +44,6 @@ and a wide range of compute accelerators such as GPU and FPGA.
 * `python` - [Download](https://www.python.org/downloads/)
 * `ninja` -
 [Download](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
-* `hwloc` version 2.3 or later (Linux only)
-  * libhwloc-dev or hwloc-devel package on linux
```

Comment:
> Update UR to statically link hwloc v2.9.3 both on linux and Windows.

If I get the change right, the dependency on `hwloc` still exists. The change removed the runtime dependency, but not the compile time dependency. Am I right?

This document describes the environment for DPC++ compiler developers, not DPC++ compiler users.

Your change seems to raise the version requirement. Please, update the `hwloc` version to 2.9.3.